src/v/cluster/id_allocator_frontend.cc

```diff
@@ -45,50 +45,88 @@ id_allocator_frontend::id_allocator_frontend(
   , _metadata_cache(metadata_cache)
   , _connection_cache(connection_cache)
   , _leaders(leaders)
-  , _controller(controller) {}
```

Comment:
adding here because github doesn't allow comment on the commit body. 

1. let's make the subject line 70-80 chars

Reply:
`len("id_allocator: add retries to wait until partition_leaders_table is populated")==76`